README.md

```diff
@@ -23,7 +23,7 @@ Deploy [CockroachDB](https://www.cockroachlabs.com/product/) on [Kubernetes](htt
 
 ## YAML with Jsonnet & CockroachDB Operator
 
-The main focus of this project is to generate all necessary Kubernetes files, like StatefulSets, Services, CronJobs, and more with [Jsonnet](https://jsonnet.org).  
+The main focus of this project is to generate all necessary Kubernetes files, like StatefulSets, Services, CronJobs, and more with [Jsonnet](https://jsonnet.org).
```

Comment:
This is actually a new line in markdown. I don't care that much though, so merging.